code/modules/clothing/suits/jobs.dm

```diff
@@ -122,51 +122,54 @@
 
 /obj/item/clothing/suit/secdressjacket
 	body_parts_covered = FULL_TORSO|ARMS
+	name = ""
```

Comment:
why is any of this like it is?

Reply:
if i remember correctly it's because byond decided that the child classes were equal to this, i'll see what i can do